open-craft/sprints
# [BB-1717] create FF and DD tickets at the end of the sprint

This implements creating FF and DD tickets automatically at the end of the sprint.

## Testing instructions 
(this will actually create new tickets, so please remember to archive them; you can see BB-1751 - BB-1756 that were created after I ran the following test)
1. Set `.env` file (you can find it in a comment on the ticket).
1. Add the following under line `sprints/dashboard/tasks.py:137`:
	```python
	cell_dict = {
	    'key': cell.key,
	    'name': cell.name,
	    'board_id': cell.board_id,
	}
	future_next_sprint = get_next_sprint(sprints, next_sprint)
	create_role_issues_task.delay(cell_dict, future_next_sprint.id, get_sprint_number(future_next_sprint))
	```
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Set frontend/.env.local file (you can find it in a comment on the ticket).
1. _Optional (this will speed up testing, as the accounts' rework is not merged into the master yet)_: remove lines 82 and 83 from `frontent/src/App.js`.
1. Navigate to frontend and run `npm i && npm start`.
1. Log in.
1. Go to the cell's dashboard and press `Complete Sprint` button.
